doc/api/esm.md

```diff
@@ -657,21 +666,26 @@ export async function resolve(specifier, context, defaultResolve) {
 }
 ```
 
-#### `getFormat(url, context, defaultGetFormat)`
+#### `load(url, context, defaultLoad)`
 
-> Note: The loaders API is being redesigned. This hook may disappear or its
-> signature may change. Do not rely on the API described below.
+> Note: Previously the experimental version had 3 separate hooks, which have
+> been consolidated into this one.
 
 * `url` {string}
 * `context` {Object}
-* `defaultGetFormat` {Function}
+  * `format` {string} The format optionally supplied by the `resolve` hook.
+  **Default:** `undefined`
+* `defaultLoad` {Function}
 * Returns: {Object}
   * `format` {string}
+  * `source` {string|SharedArrayBuffer|Uint8Array}
+
+The `load` hook provides a way to define a custom method of determining how
+a URL should be interpreted, retrieved, and parsed.
 
-The `getFormat` hook provides a way to define a custom method of determining how
-a URL should be interpreted. The `format` returned also affects what the
-acceptable forms of source values are for a module when parsing. This can be one
-of the following:
+For chained loaders, intermediary values of `format` can be any string (perhaps
+to be used by another loader in the chain); however, the _final_ value of
+`format` must be one of the following:
```

Comment:
The current implementation doesn't allow loader chaining does it? I think this needs to be rephrased.

Reply:
Correct: Not currently. This will be true when it does, so it's not applicable yet. I could remove this bit until that gets added